Update Digital Design Chap04.md | 数逻第四章修改

YardRat0117 opened the pull request:
Added some further discussion about SR Master-Slave Flip-Flop 1's catching issue in Digital Design Chapter 04 notes.
关于 SR 主仆触发器的 1‘s catching 问题增加了一些深入讨论（数逻第四章笔记），可能有点主观，第一次做这种笔记的public contribution，麻烦isshiki review了

唉唉我接着补天去

YardRat0117 committed:
Update Chap04.md (#1)

Added some further discussion about SR Master-Slave Flip-Flop 1's catching issue in Digital Design Chapter 04 notes.

IsshikiHugh commented:
笑死，具体内容我已经忘光了，已经看不懂这些是什么东西了x
我大概浏览了一下，主要是引入了一些新的之前没有定义过的特殊词汇，可能让人有点一头雾水。
可以麻烦你尝试着不用这些特殊词汇解释一下吗x
另外，你可以把几个英文词汇翻译一下吗，这样读下来观感一致一点（或者你可以用我之前那种中英文都标的形式）。

感谢贡献~

IsshikiHugh reviewed (changes requested):
“这种非原子性是一种 inherent 的性质，无法真正 eliminate 掉”，这句话感觉可以更直白一点。
然后感觉这句话和后面几句话的逻辑联系可能没那么强？我的意思是我感觉这句话作为这一小段的第一句话是不是没有起到开宗明义的目的呀，你看看？

“但在实际工程中已能充分满足系统对稳定性的要求”，我忘了有没有这个结论了，如果不是课内提到的话，可以麻烦你加个 reference 吗，方便想细节了解的同学点进去看看更细节的东西~

YardRat0117 commented:
OK, gonna work on this later on, sounds quite much of  workload. 
现在先补天

YardRat0117 commented:
Good news, currently working on a further, full illustration, and with more plain narratives.
Gotta publish it to my own homepage (it's pure English and too long, thus not suitable for here).
And then I guess I'd try to do some simple and concrete Chinese illustration here. 
:)

YardRat0117 committed:
discussion on 1s-catching updated

- More readable and easy-to-understand discussion.

- Additional links for reference.

YardRat0117 committed:
Polished Expression
